packages/wonder-blocks-tabs/src/components/navigation-tab-item.tsx

```diff
@@ -0,0 +1,81 @@
+import {addStyle} from "@khanacademy/wonder-blocks-core";
+import {CSSProperties, StyleSheet} from "aphrodite";
+import * as React from "react";
+import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
+import {semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+
+type Props = {
+    /**
+     * The contents of the NavigationTabItem (usually a Link component)
+     */
+    children: React.ReactElement;
+};
+
+const StyledLi = addStyle("li");
+
+/**
+ * A component for a tab item in NavigationTabs. It is used with a Link
+ * component.
+ *
+ * ## Usage
+ *
+ * ```jsx
+ * import {NavigationTab, NavigationTabItem} from "@khanacademy/wonder-blocks-tabs";
+ * import Link from "@khanacademy/wonder-blocks-link";
+ *
+ * <NavigationTabs>
+ *  <NavigationTabItem>
+ *    <Link href="/link-1">Link 1</Link>
+ *  </NavigationTabItem>
+ *  <NavigationTabItem>
+ *    <Link href="/link-2">Link 2</Link>
+ *  </NavigationTabItem>
+ * </NavigationTabs>
+ * ```
+ */
+export const NavigationTabItem = (props: Props) => {
+    const {children} = props;
+
+    function renderChildren() {
+        const linkProps = {
+            style: [typographyStyles.Body, styles.link],
+        };
+
+        return React.cloneElement(children, linkProps);
+    }
+
+    return <StyledLi style={styles.root}>{renderChildren()}</StyledLi>;
+};
+
+const underlineStyles: CSSProperties = {
+    content: '""',
+    display: "block",
+    position: "absolute",
+    bottom: 0,
+    left: 0,
+    right: 0,
+    height: sizing.size_050,
+};
+
+const styles = StyleSheet.create({
+    root: {
+        listStyle: "none",
+        display: "inline-flex",
+    },
+    link: {
+        color: semanticColor.text.primary,
+        paddingBlock: sizing.size_150,
+        paddingInline: 0,
+        position: "relative",
+        ":hover": {
+            textDecoration: "none",
+            [":after" as any]: {
+                // We use :after to apply underline styles instead of textDecoration
+                // so the underline is shown under icons in Link too
+                ...underlineStyles,
+                backgroundColor:
+                    semanticColor.action.primary.progressive.hover.border,
+            },
```

Comment:
I'd like to propose another approach to get your thoughts on:

Do you think `boxShadow` could work in this case?

e.g.
```
boxShadow: inset 0 -${sizing.size_050} 0 0 ${semanticColor.action.primary.progressive.hover.border}
```

If you think that works, this could simplify the `:after` approach. Also would play nice with the idea we have for changing global focus to use `box-shadow` instead of `outline` :)

Reply:
@beaesguerra After discussing offline, we have found that this needs to be revisited in the next PR (or the PR that tackles styling). Probably `boxShadow` would still be suitable based on the latest design specs.